src/CitizenProject/CitizenProjectManager.php

```diff
@@ -323,4 +329,21 @@ public function findAdherentNearCitizenProjectOrAcceptAllNotification(CitizenPro
     {
         return $this->getAdherentRepository()->findByNearCitizenProjectOrAcceptAllNotification($citizenProject, $offset, $excludeSupervisor, $radius);
     }
+
+    public function removeImage(CitizenProject $citizenProject): bool
+    {
+        try {
+            // Delete image from cloud storage
+            $this->storage->delete($citizenProject->getImagePath());
+            $this->glide->deleteCache($citizenProject->getImagePath());
+
+            $citizenProject->setImageUploaded(false);
+            $this->getManager()->persist($citizenProject);
+            $this->getManager()->flush();
+        } catch (\Exception $e) {
```

Comment:
It's very dangerous to catch all exceptions like this. It can hide a relevant error. What is the intention here ?